main.go

```diff
@@ -20,17 +20,17 @@ const (
 	usageString = `Usage: license [flags] [license-type]
 
 Flags:
-       -help     print help information
-       -list     print list of available license types
-   -n, -name     full name to use on license (default %q)
-   -o, -output   path to output file (prints to stdout if unspecified)
-       -version  print version
-   -y, -year     year to use on license (default %q)
+	-help     print help information
+	-list     print list of available license types
+	-n, -name     full name to use on license (default %q)
+	-o, -output   path to output file (prints to stdout if unspecified)
+	-v, -version  print version
+	-y, -year     year to use on license (default %q)
```

Comment:
This prints:
```
Flags:
	-help     print help information
	-list     print list of available license types
	-n, -name     full name to use on license (default "Nishanth Shanmugham")
	-o, -output   path to output file (prints to stdout if unspecified)
	-v, -version  print version
	-y, -year     year to use on license (default "2021")
```

Let's align the flag descriptions on the same column, like before:
```
Flags:
	-help         print help information
	-list         print list of available license types
	-n, -name     full name to use on license (default "Nishanth Shanmugham")
	-o, -output   path to output file (prints to stdout if unspecified)
	-v, -version  print version
	-y, -year     year to use on license (default "2021")
```

Reply:
Good catch, I've fixed this now. 👍